package.json

```diff
@@ -67,6 +67,6 @@
     "eventemitter3": "^3.1.0",
     "ramda.clone": "^0.26.1",
     "script-loader": "^0.7.0",
-    "thrift": "^0.10.0"
+    "thrift": "apache/thrift#v0.13.0"
```

Comment:
Note, this will only affect the Node.js connector - the browser connector runtime is still inlined at https://github.com/omnisci/mapd-connector/blob/master/thrift/browser/thrift.js for now.